util/automation.py

```diff
@@ -32,8 +32,12 @@
 
 DEEPSPEECH_CLONE_PATH  = os.path.abspath(os.environ.get('ds_clone_path', './ds_exec_clone/'))
 
-CACHE_DIR = os.path.join(BaseDirectory.xdg_cache_home, 'deepspeech_wer')
-DATA_DIR  = os.path.join(BaseDirectory.xdg_data_home, 'deepspeech_wer')
+# Cache and data should be used for small amounts of data
+CACHE_DIR     = os.path.join(BaseDirectory.xdg_cache_home, 'deepspeech_wer')
+DATA_DIR      = os.path.join(BaseDirectory.xdg_data_home, 'deepspeech_wer')
```

Comment:
I think having a separate LOG_DIR or similar would be nicer than having DATA_DIR and HUGE_DATA_DIR. At what point is the 'huge' distinction made? And why can't smaller data go into HUGE_DATA_DIR anyway? It's related, does it make sense to have this divide?